osu.Game.Rulesets.Taiko/Objects/Drawables/DrawableHit.cs

```diff
@@ -42,7 +42,7 @@ protected override void CheckForResult(bool userTriggered, double timeOffset)
             }
 
             var result = HitObject.HitWindows.ResultFor(timeOffset);
-            if (result == HitResult.None)
+            if (result <= HitResult.Miss)
```

Comment:
Was this a conscious change? This makes objects un-missable if hit early.

Reply:
It's the same in normal taiko, it's intended

Reply:
You can do as much hits as you want on empty space, so misses aren't being accounted for

Reply:
That's not true: https://streamable.com/tkdqn

Reply:
Granted, the miss range should be equivalent of `HitResult.Meh` in this case, rather than the (400, 400, 400) that it's using (and `Meh` itself shouldn't exist for taiko).

Reply:
Changed TaikoHitWindows